tests/test_onco_print.py

```diff
@@ -73,7 +74,29 @@ def oncoprint_props_callback(
     return typed_prop_value
 
 
-# below are tests for changing the props of the React component
+# Tests for layout props (under "Customize" tab)
+def test_showoverview(dash_threaded):
+    """Test the overview display."""
+    def assert_callback(prop_value, nclicks, input_value):
+        answer = ''
+        if nclicks is not None:
+            if PROP_TYPES['bool'](input_value) == prop_value:
+                answer = 'PASSED'
```

Comment:
In [the `test_skeleton` file](https://github.com/plotly/dash-bio/blob/e16f5405bc052d70fbb1c2da52969c94ab55e709/tests/test_skeleton.py#L22), we have this: 

```
# pass/fail strings
PASS = 'PASSED'
FAIL = 'FAILED'
```

Do you think it's a good idea to include this? @Bachibouzouk is an advocate of "hard-coding" strings like this to avoid errors that could pop up if someone makes a typo.

Reply:
Good catch. I kept these variables in `test_onco_print.py` and I'm not making use of them, so it's not consistent... I would be in favour of using the `PASS` variable.